### ISSUE TITLE
Not compatible with the latest version of Cordova

### ISSUE THREAD

**Issue by jbdoster** [jcesarmobile/FilePicker-Phonegap-iOS-Plugin#57]
## Description
Trying to use "ionic cordova platform add ios" and receiving error

## Information
Error received:
```
Finding scripts for "before_plugin_install" hook from plugin cordova-plugin-filepicker on ios platform only.
No scripts found for hook "before_plugin_install".
Install start for "cordova-plugin-filepicker" on ios.
Beginning processing of action stack for ios project...
Action stack processing complete.
pods.json found in platforms/ios
Podfile found in platforms/ios
Install complete for cordova-plugin-filepicker on ios.
Finding scripts for "after_plugin_install" hook from plugin cordova-plugin-filepicker on ios platform only.
Executing script found in plugin cordova-plugin-filepicker for hook "after_plugin_install": plugins/cordova-plugin-filepicker/src/ios/hooks/install_entitlements.js
Running hook to add iCloud entitlements
Failed to install 'cordova-plugin-filepicker': CordovaError: Using "requireCordovaModule" to load non-cordova module "xcode" is not supported. Instead, add this module to your dependencies and use regular "require" to load it.
    at Context.requireCordovaModule (/Users/dev/.npm-global/lib/node_modules/cordova/node_modules/cordova-lib/src/hooks/Context.js:57:15)
    at module.exports (/Users/dev/Documents/w+p/whisker-paw-mobile/plugins/cordova-plugin-filepicker/src/ios/hooks/install_entitlements.js:8:23)
    at runScriptViaModuleLoader (/Users/dev/.npm-global/lib/node_modules/cordova/node_modules/cordova-lib/src/hooks/HooksRunner.js:181:32)
    at runScript (/Users/dev/.npm-global/lib/node_modules/cordova/node_modules/cordova-lib/src/hooks/HooksRunner.js:157:16)
    at /Users/dev/.npm-global/lib/node_modules/cordova/node_modules/cordova-lib/src/hooks/HooksRunner.js:125:20
    at process._tickCallback (internal/process/next_tick.js:68:7)
CordovaError: Using "requireCordovaModule" to load non-cordova module "xcode" is not supported. Instead, add this module to your dependencies and use regular "require" to load it.
    at Context.requireCordovaModule (/Users/dev/.npm-global/lib/node_modules/cordova/node_modules/cordova-lib/src/hooks/Context.js:57:15)
    at module.exports (/Users/dev/Documents/w+p/whisker-paw-mobile/plugins/cordova-plugin-filepicker/src/ios/hooks/install_entitlements.js:8:23)
    at runScriptViaModuleLoader (/Users/dev/.npm-global/lib/node_modules/cordova/node_modules/cordova-lib/src/hooks/HooksRunner.js:181:32)
    at runScript (/Users/dev/.npm-global/lib/node_modules/cordova/node_modules/cordova-lib/src/hooks/HooksRunner.js:157:16)
    at /Users/dev/.npm-global/lib/node_modules/cordova/node_modules/cordova-lib/src/hooks/HooksRunner.js:125:20
    at process._tickCallback (internal/process/next_tick.js:68:7)
Using "requireCordovaModule" to load non-cordova module "xcode" is not supported. Instead, add this module to your dependencies and use regular "require" to load it.
[ERROR] An error occurred while running subprocess cordova.
        
        cordova platform add ios --verbose --save exited with exit code 1.
        
        Re-running this command with the --verbose flag may provide more information.
  ionic:utils-process onBeforeExit handler: process.exit received +0ms
  ionic:utils-process onBeforeExit handler: running 1 functions +0ms
  ionic:utils-process onBeforeExit handler: exiting (exit code 1) +148ms
```

### Command or Code
ionic cordova platform add ios

### Environment, Platform, Device
macOS High Sierra
Version 10.13.6
Processor 2.2 GHz Intel Core i7

### Version information
Full output from command cordova info [here](https://github.com/jbdoster/cordova-info/blob/master/output)

**Comment by jcesarmobile** [comment]
See https://github.com/jcesarmobile/FilePicker-Phonegap-iOS-Plugin/issues/56

**Comment by jcesarmobile** [comment]
As an user has send a [PR](https://github.com/jcesarmobile/FilePicker-Phonegap-iOS-Plugin/pull/58) I've published 1.1.6 with the fix
